js/main.js

```diff
@@ -1,2 +1,3 @@
-// import './util.js';
-// import './mocks.js';
+import './util.js';
+import './mocks.js';
+import './sample';
```

Comment:
Импортируем только нужные файлы
```suggestion
import './sample.js';
```

Reply:
Лучше переименовать
```suggestion
import './previews.js';
```